var/spack/repos/builtin/packages/py-pint/package.py

```diff
@@ -27,11 +29,13 @@ class PyPint(PythonPackage):
     version("0.9", sha256="32d8a9a9d63f4f81194c0014b3b742679dce81a26d45127d9810a68a561fe4e2")
     version("0.8.1", sha256="afcf31443a478c32bbac4b00337ee9026a13d0e2ac83d30c79151462513bb0d4")
 
-    depends_on("python@3.8:", type=("build", "run"), when="@0.19:")
-    depends_on("py-setuptools@41:", when="@0.16:", type="build")
+    depends_on("python@3.9:", when="@0.22:", type=("build", "run"))
+    depends_on("python@3.8:", when="@0.19:0.21", type=("build", "run"))
+    depends_on("py-setuptools@61:", when="@0.21:", type="build")
+    depends_on("py-setuptools@41:", when="@0.16:0.20", type="build")
     depends_on("py-setuptools@41:", when="@0.11:0.15", type=("build", "run"))
     depends_on("py-setuptools", when="@:0.10", type=("build", "run"))
     depends_on("py-setuptools-scm@3.4.3:+toml", when="@0.11:", type="build")
     depends_on("py-setuptools-scm", when="@0.10", type="build")
-    depends_on("py-packaging", type=("build", "run"), when="@0.13:18")
-    depends_on("py-importlib-metadata", type=("build", "run"), when="@0.13:18 ^python@:3.7")
+    depends_on("py-packaging", when="@0.13:18", type=("build", "run"))
+    depends_on("py-importlib-metadata", when="@0.13:18 ^python@:3.7", type=("build", "run"))
```

Comment:
Missing new dep on typing-extensions

Reply:
Thanks, I missed that one. Perhaps someday `spack checksum` will read the `.toml` files to generate dependencies? 😬

Reply:
And perhaps someday everyone will use `.toml` files 😆